ci/release/update-version.sh

```diff
@@ -19,41 +19,41 @@
 
 
 ## Usage
-# bash update-version.sh <type>
-#     where <type> is either `major`, `minor`, `patch`
+# bash update-version.sh
+# Promotes patch version, version format : v<Year>.<Month>.<patch>
 
 set -e
 
-# Grab argument for release type
-RELEASE_TYPE=$1
+# Overwrite argument value to always trigger a patch release
+RELEASE_TYPE="patch"
 
 # Get current version and calculate next versions
-CURRENT_TAG=`git tag | grep -xE 'v[0-9\.]+' | sort --version-sort | tail -n 1 | tr -d 'v'`
+CURRENT_TAG=`git tag | grep -xE "v$(date '+%Y\.%m')\.[0-9]+" | sort --version-sort | tail -n 1 | tr -d 'v'`
+# In case no tag exists for this month
+if [ -z "${CURRENT_TAG}" ]; then
+  CURRENT_TAG="$(date '+%Y.%m').-1"
```

Comment:
why do we use `-1` here?

Reply:
If no tags are found in the same month, it means this is the first release of this month- the next version (v2021.02.0) will be set in: `NEXT_PATCH=$((CURRENT_PATCH + 1))`. I could wrap the next commands in an if block instead.